x/delayedack/keeper/rollapp_packet.go

```diff
@@ -65,12 +65,15 @@ func (k Keeper) UpdateRollappPacketTransferAddress(
 	}
 	// Set the recipient and sender based on the rollapp packet type
 	recipient, sender := transferPacketData.Receiver, transferPacketData.Sender
+	originalRecipient := rollappPacket.OriginalRecvReceiver // can be empty
```

Comment:
why is this line necessary vs just initializing `originalRecipient`? seems to me like the `originalRecipient` will always be empty after that.

Reply:
yeah seems simpler to have empty string 👍  will change